test/sql/function/list/aggregates/histogram.test

```diff
@@ -55,26 +55,38 @@ SELECT list_histogram(['2021-08-20'::TIMESTAMP])
 ----
 {2021-08-20 00:00:00=1}
 
+# timestamp_s
+query I
+SELECT list_histogram(['2021-08-20'::TIMESTAMP_S])
+----
+{2021-08-20 00:00:00=1}
+
+# timestamp_ms
+query I
+SELECT list_histogram(['2021-08-20'::TIMESTAMP_MS])
+----
+{2021-08-20 00:00:00=1}
+
+# timestamp_ns
+query I
+SELECT list_histogram(['2021-08-20'::TIMESTAMP_NS])
+----
+{2021-08-20 00:00:00=1}
+
 # timestamp with time zone
 query I
 SELECT list_histogram(['2021-08-20'::TIMESTAMPTZ])
 ----
 {2021-08-20 00:00:00+00=1}
 
-mode skip
-
-# FIXME: test cases for the types date, time, timetz, timestamp_s, timestamp_ms, timestamp_ns, interval
-# currently fail due to a bug in the histogram code
-# they should be added once that bug is resolved
-
-foreach type date time timestamp_s timestamp_ms timestamp_ns timetz
+# types date, time, timetz
```

Comment:
Can we separate the tests here? The casts will just fail for the time types because of syntax errors.  its also pretty easy to add the missing histogram variants (I just did the same thing for the different TS scales).